com.unity.ml-agents/Runtime/SideChannels/EngineConfigurationChannel.cs

```diff
@@ -27,13 +27,14 @@ public override void OnMessageReceived(IncomingMessage msg)
             var qualityLevel = msg.ReadInt32();
             var timeScale = msg.ReadFloat32();
             var targetFrameRate = msg.ReadInt32();
+            var captureFrameRate = msg.ReadInt32();
```

Comment:
Might want to put defaults in these; this will probably "read" 0 since it's going past the end of the data.